lib/themes/theme.d.ts

```diff
@@ -171,6 +171,7 @@ type SpacingVariants =
   | 'xlarge'
   | 'xxlarge';
 type HorizontalSpacingVariants = SpacingVariants | 'gutter';
+type VerticalPaddingVariants = SpacingVariants | 'standardTouchableText';
```

Comment:
Didn't we decide on these types being singular?

Reply:
Fixed. Also updated the other two spacing types to be consistent.